### ISSUE TITLE
Proxifié certaines route de l'api de LXD (websocket

### ISSUE THREAD

**Issue by Yghore** [Luquor/VirtUI-backend#3]
Pour permettre l'utilisation des websockets (terminal, execution de commande, ...), notre API a besoin de proxifié certaines parties de l'api de LXD pour permettre une communication

**Comment by Luquor** [comment]
Ça pourrait être quelques chose du genre ou pas ?

`
    r := chi.NewRouter()
    r.Use(proxy.ForwardedHeaders(
        proxy.NewForwardedHeadersOptions().
            WithForwardLimit(2).
            ClearTrustedProxies().AddTrustedProxy("10.0.0.1"),
    ))
`

**Comment by Yghore** [comment]
Non, je crois que ça, sa sert a faire fonctionner chi, si il est proxifié (et dcp il accepte seulement le proxy 10.0.0.1), nous il faut que certaines routes tape sur LXD Api directement (utiliser HaProxy ou Nginx)
